timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/dropwizard/endpoints/GraphQl.java

```diff
@@ -56,6 +79,7 @@ public Response post(String query, @HeaderParam("accept") String acceptHeader, @
     return executeGraphQlQuery(query, userId, dataSet);
   }
 
+
   public boolean hasSpecifiedAcceptHeader(@HeaderParam("accept") String acceptHeader) {
     return acceptHeader == null || acceptHeader.isEmpty() || "*/*".equals(acceptHeader);
```

Comment:
It looks like the method returns the opposite to what is says. It returns true when no specific acceptHeader is specified.